tests/Performance/Locust/ResourceStatusPerformance.py

```diff
@@ -0,0 +1,60 @@
+import time
```

Comment:
Please make this python 3 compatible. 
Start by adding `from __future__ import print_function` and change the print below.